service/awsconfig/v10/resource/lifecycle/create.go

```diff
@@ -146,7 +146,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	{
 		r.logger.LogCtx(ctx, "level", "debug", "message", "start inspection of node configs for the guest cluster")
 
-		n := v1.NamespaceDefault
+		n := v1.NamespaceAll
```

Comment:
When listing we go with all namespaces to get the full list for inspection below.